driver_cpl/bld/build-namelist

```diff
@@ -287,14 +287,11 @@ foreach my $file (@files) {
     my @e = $xml->elements_by_name('entry');
     while ( my $e = shift @e ) {
 	my %a = $e->get_attributes();
+	my $val = $a{value};
```

Comment:
This code was changed because after I fixed the resolve issue in the env_run.xml this was causing the ATM2ICE_SMAPNAME and others to get unresolved values which they then tried to resolve incorrectly.
Moving DIN_LOC_ROOT to after the expand_xml_var solved this problem.